### DISCUSSION TITLE
A new use for XP

### DISCUSSION THREAD

**Discussion by Notarin** [PrideCraft/Public-Issue-Tracker#10] [Ideas]
Just had a great Idea for XP, a command, that when toggled on, Will do some math to calculate, for the amount of xp you have, anytime you take damage, it will be reduced. and then it will remove a portion of xp.
High xp amounts will block a large amount of damage, but drain the xp faster.
Low xp will block low damage but drain slower.
And this will be opt in. Hence the xp-gaurd command

**Comment by basil222** [comment]
I think that this should stay as a option, people can active if they want, since it can consume a lot of xp

**Reply by Notarin** [reply]
Oh yeah, it'll definitely be toggle-able
